src/main/java/org/folio/circulation/domain/ItemStatus.java

```diff
@@ -14,7 +14,8 @@
   MISSING("Missing"),
   PAGED("Paged"),
   ON_ORDER("On order"),
-  IN_PROCESS("In process");
+  IN_PROCESS("In process"),
+  DECLARED_LOST("Declared lost");
```

Comment:
What impact does this new status have? 

Can requests be made when an item is declared lost? 

Can the item be checked out? 

What happens if it is checked in?

Can it be marked lost in the inventory UI?

Reply:
It is the new status of an item according to the requirements.

Reply:
I understand that the new status is needed for the work. When a new status is introduced, it is necessary to consider the impact on the system.

My questions above are part of that. There likely need to be tests for these situations (and for creating requests for declared lost items).

If you don’t know what the answers are to these questions, maybe the PO needs to be involved in exploring those requirements.

Reply:
These questions have been forwarded to the PO. In what way, do you think, the answers might affect the current implementation?